Update release notes for #27043 (#28088)
gradle/gradle: platforms/documentation/docs/src/docs/release/notes.md

Using the new `--no-comments` option allows generating projects that contain only code without clarifying comments.
The resulting build files and source files are smaller and less noisy.

#### Emitting separate problems for each issue found in a TOML catalog file

> When a TOML catalog files are used with invalid TOML syntax every distinct problem is emitted separately via the Problems API.

Reviewer: ```suggestion
When TOML catalog files with invalid TOML syntax are used, each distinct problem is reported separately using the Problems API.
```

Would it be possible to include a screenshot or a snapshot of the report?

Author: It's just an even we send, what it looks like in a report entirely depends on the consumer e.g. the IDEs.

Reviewer: Sounds good, no screenshot, just the grammar update and we are good to go.
